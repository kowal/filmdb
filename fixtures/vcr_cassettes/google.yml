---
http_interactions:
- request:
    method: get
    uri: http://google.pl/
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept:
      - ! '*/*'
      User-Agent:
      - Ruby
  response:
    status:
      code: 301
      message: Moved Permanently
    headers:
      Location:
      - http://www.google.pl/
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Mon, 20 May 2013 20:27:44 GMT
      Expires:
      - Wed, 19 Jun 2013 20:27:44 GMT
      Cache-Control:
      - public, max-age=2592000
      Server:
      - gws
      Content-Length:
      - '218'
      X-Xss-Protection:
      - 1; mode=block
      X-Frame-Options:
      - SAMEORIGIN
    body:
      encoding: US-ASCII
      string: "<html><head></head><body>MockedPageContent</body></html>"
    http_version: 
  recorded_at: Mon, 20 May 2013 20:27:43 GMT
recorded_with: VCR 2.4.0
