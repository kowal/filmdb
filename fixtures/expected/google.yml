---
- "<html><head><metahttp-equiv=\"content-type\"content=\"text/html;charset=us-ascii\"></head><body>mockedpagecontent</body></html>"